listener/listener.go

```diff
@@ -63,89 +63,130 @@ func (l *listener) ProcessEvent(w http.ResponseWriter, request *http.Request) {
 		log.Error("error reading webhook event", zap.Error(err))
 		return
 	}
+
+	// only process events that occur when scanning is finished
+	if event.Type != harbor.SCANNING_FAILED && event.Type != harbor.SCANNING_COMPLETED {
```

Comment:
Probably on this already, but now that we're only listening for these two events, we can drop [`PUSH_ARTIFACT` from the demo](https://github.com/rode/demo/blob/cc937553b24c950c1f1d7a0a4a84c3e957e9872b/tf/modules/harbor-config/main.tf#L9)